packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form.feature

```diff
@@ -0,0 +1,107 @@
+      | EUR      | Canada                                | Beneficiary Account Name, Bank Account Number, SWIFT Code                                            |
+      | EUR      | Spain, Ireland, United Kingdom, Japan | Beneficiary Account Name, IBAN, SWIFT Code                                                           |
+      | JPY      | Japan                                 | Beneficiary Account Name, Bank Account Type, Bank Account Number, SWIFT Code, Bank Name, Branch Code |
+    # According to mock data currently available
+
+  Scenario Outline: Bank country dropdown list is dynamic depending on the partner's currency
+    Given the bank account payment method is selected
+    And the selected currency is <currency>
+    Then the bank country options are <countries>
+
+    Examples: 
+      | currency | countries                                                    |
+      | USD      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | GBP      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | AUD      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | CAD      | Canada                                                       |
+      | EUR      | United States, Canada, Spain, Ireland, United Kingdom, Japan |
+      | JPY      | Japan                                                        |
+
+  Scenario Outline: Payment Method text is dynamic depending on the currency and country selected
+    Given the bank account payment method is selected
+    And the selected currency is <currency>
+    And the selected bank country is one of <bankCountry>
+    Then <paymentMethodText> is shown
+    And the withdrawal fee is shown
+
+    Examples: 
+      | currency | bankCountry                                                  | paymentMethodText |
+      | USD      | United States                                                | EFT Withdrawal    |
+      | USD      | Canada, Spain, Ireland, United Kingdom, Japan                | FX Wire           |
+      | GBP      | United Kingdom                                               | EFT Withdrawal    |
+      | GBP      | United States, Canada, Spain, Ireland, Japan                 | FX Wire           |
+      | AUD      | United States, Canada, Spain, Ireland, United Kingdom, Japan | FX Wire           |
+      | CAD      | Canada                                                       | EFT Withdrawal    |
+      | EUR      | Spain, Ireland,  United Kingdom                              | EFT Withdrawal    |
+      | EUR      | United States, Canada, Japan                                 | FX Wire           |
+      | JPY      | Japan                                                        | EFT Withdrawal    |
+    # Not sure what the actual specs for when it's shown are yet
+
+  Scenario Outline: PayPal option is dynamically shown
```

Comment:
This should include a currency that isn't supported by Paypal so we are testing that it also doesn't show up when it shouldn't